ansible/deploy.sh

```diff
@@ -16,4 +16,4 @@
 
 inventory=${INVENTORY:-inventory}
 
-ansible-playbook -i ${inventory} cluster.yml $@
+ansible-playbook -i ${inventory} deploy.yml $@
```

Comment:
``deploy-cluster.yml`` would be more appropriate. Although, in future the deployment scripts will get migrated under ``kubernetes/kube-deploy``, ansible scripts can provide additional functionality, e.g. collection of logs from cluster nodes.